scripts/clone_project.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+clone_project()
+{
+  if [ -z "$2" ];
+  then
+    printf "Empty paths are dangerous in use. Use another path instead for submodules installation or update.\n"
+    exit 1
+  fi
+  if [[ ! -d "$2" || "$1" == "--update" ]]; then
```

Comment:
Arguments usage is not valid here. Please, resee each argument using in this function